.ci/pitest.sh

```diff
@@ -222,6 +222,7 @@ pitest-ant)
 pitest-blocks)
   mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
   declare -a ignoredItems=(
+  "RightCurlyCheck.java.html:<td class='covered'><pre><span  class='survived'>                    &#38;&#38; shouldBeAloneOnLineWithAloneOrSinglelineOption(details, targetSrcLine)</span></pre></td></tr>"
```

Comment:
Basically same reasoning as in #6530.
The code in question this time is
```java
return bracePolicy == RightCurlyOption.ALONE
            && shouldBeAloneOnLineWithAloneOption(details, targetSrcLine)
        || bracePolicy == RightCurlyOption.ALONE_OR_SINGLELINE
            && shouldBeAloneOnLineWithAloneOrSinglelineOption(details, targetSrcLine)
        || bracePolicy == RightCurlyOption.SAME
            && details.shouldCheckLastRcurly
            && shouldBeAloneOnLineWithAloneOrSinglelineOption(details, targetSrcLine);
```
and while pitest report again reports one line too early, the surviving mutant is replacing `bracePolicy == RightCurlyOption.SAME` by `true`.
This means to make a difference `bracePolicy` needs to be either `ALONE` or `ALONE_OR_SINGLELINE`, so that in bug-free code the condition would have been `false`.
If the option is `ALONE`, `shouldBeAloneOnLineWithAloneOption` has to be `false` or the third `||`-part is not even evaluated.
If the option is `ALONE_OR_SINGLELINE`, `shouldBeAloneOnLineWithAloneOrSinglelineOption` has to be `false` or the third `||`-part is not even evaluated.
`shouldBeAloneOnLineWithAloneOption` is effectively a subset of `shouldBeAloneOnLineWithAloneOrSinglelineOption`, so if first is `false`, second is also false.

This means again, when the third `||`-part is evaluated, its third `&&`-part is always `false` and so the first `&&`-part which is the mutant survivor does not make any difference.
It is basically only there for better code readability and stability for further changes, for example if another `RightCurlyOption` is added.
Same options as for #6530.